lib/trainer/options.rb

```diff
@@ -33,7 +33,13 @@ def self.available_options
                                      env_name: "TRAINER_FAIL_BUILD",
                                      description: "Should this step stop the build if the tests fail? Set this to false if you're handling this with a test reporter",
                                      is_string: false,
-                                     default_value: true)
+                                     default_value: true),
+        FastlaneCore::ConfigItem.new(key: :xcpretty_naming,
+									 short_option: "-x",
```

Comment:
Looks like we have some inconsistent indentation. Try `rubocop -a` to fix these with the linter.

Reply:
Thanks for the command ! Fixed it.